browser/net/url_context.cc

```diff
@@ -114,16 +113,15 @@ void BraveRequestInfo::FillCTXFromRequest(const net::URLRequest* request,
   ctx->referrer = GURL(request->referrer());
   ctx->referrer_policy = request->referrer_policy();
 
-  auto* request_info = content::ResourceRequestInfo::ForRequest(request);
-  if (request_info) {
-    ctx->resource_type = request_info->GetResourceType();
-    if (auto* context = request_info->GetContext()) {
-      ctx->is_webtorrent_disabled = IsWebTorrentDisabled(context);
-    }
-  }
+  // auto* request_info = content::ResourceRequestInfo::ForRequest(request);
+  // if (request_info) {
+  //   ctx->resource_type = request_info->GetResourceType();
+  //   if (auto* context = request_info->GetContext()) {
+  //     ctx->is_webtorrent_disabled = IsWebTorrentDisabled(context);
```

Comment:
some tests should fail, right? I hope I'll be able to quickly jump in with the fix.

Reply:
all tests passed on the last run

Reply:
Not sure if this is good for us :D